docs/serving/autoscaling/scale-bounds.md

```diff
@@ -157,23 +157,17 @@ When the Revision is created, the larger of initial scale and lower bound is aut
           allow-zero-initial-scale: "true"
     ```
 
+## Stable window
 
+The stable window defines minimal time that will pass between descisions to change the scale of a revision.
```

Comment:
Is this true? What if we're in panic mode?

Reply:
you are right, I will add reference to panic mode.